docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -883,6 +886,83 @@ quarkus.http.auth.permission.http-upgrade.paths=/end
 quarkus.http.auth.permission.http-upgrade.policy=authenticated
 ----
 
+=== Secure server endpoints with permission checkers
+
+WebSocket endpoints can be secured with the xref:security-authorize-web-endpoints-reference.adoc#permission-checker[permission checkers].
+We strongly recommend to revise the <<secure-http-upgrade>> section and secure the HTTP upgrade like in the example below:
```

Comment:
What exactly does this sentence mean? Should a user always use `@PermissionsAllowed`?

Reply:
I tried to say to users: if you don't need to secure payload, you should secure HTTP upgrade. I know you have example above that allows to secure `@OnTextMessage` with `@RolesAllowed` and handle failure, but I don't believe it is much practical, because unless you close the connection, you will have connection for endpoint that can't be used. Maybe I should rephrase this. Suggestions?

Reply:
>  Should a user always use @PermissionsAllowed

any standard security annotation is fine

Reply:
> I wonder if we should move the HTTP upgrade section before the @OnTextMessage with @RolesAllowed example?

yes, I'll do it and try to rewrite the sentence as well.

Reply:
moved the sections and adjusted text, please have a look